tests/js/ecosystem.test.js

```diff
@@ -13,7 +13,75 @@
 
 import { expect, test } from "@playwright/test";
 
-import { click, setMobile } from "./utils";
+import { click, getPosition, setMobile, setDesktop, scrollDown } from "./utils";
+
+test.describe("Reverted Qiskit top nav bar does not break positioning of", () => {
+  test("Furo's menu bars when scrolled down", async ({ page }) => {
+    await page.goto("sphinx_guide/lists.html");
+
+    const check = async () => {
+      const pageToCPosition = await getPosition(
+        page,
+        "div.toc-title-container",
+      );
+      expect(pageToCPosition).toEqual({ x: 1344, y: 0 });
```

Comment:
Where does the `1344` come from? Same for other values (e.g. L61, L66, etc.)
Might be good to add a comment so people know when it's appropriate to change these values.

Reply:
As a human, I checked first that the site renders how I like. Then, I let the test run and see what the value should be.

Good idea on a comment.

Reply:
Ok gotcha, where can we find the value? Will it appear in failed test results?

Reply:
Yes, exactly. I added a comment.